### PR TITLE
chore: disabling dependabot version updates

### PR DESCRIPTION
**Pull request by zakir-code**
## Summary by CodeRabbit

- **Chores**
  - Updated the schedule for dependency updates to run weekly on Fridays.
  - Adjusted the limit for open pull requests related to dependency updates.

### CODE REVIEW

**Comment by zakir-code** [comment]
Ref: https://github.com/FunctionX/fx-core/blob/main/.github/dependabot.yml

**Comment by zakir-code** [comment]
Ref: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates

File: .github/dependabot.yml [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
+      interval: "weekly"
+      day: "monday"
+      time: "09:00"
+    open-pull-requests-limit: 0
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "friday"
+      time: "10:00"
```
Comment (zakir-code) [inline .github/dependabot.yml:20]:
```suggestion
      time: "09:00"
```

**Review by fx0x55 — APPROVED** [review]
mg

**Comment by fx0x55** [comment]
mg

**Pull request merged**